conf/40-escape

```diff
@@ -2,6 +2,7 @@
 # Make zsh exit on escape
 # © 2013 Axel Wagner and contributors (see also: LICENSE)
 function _shellex_exit {
+    rm $SHELLEX_SIZE_FILE
```

Comment:
Why is this needed? Doesn't the bouncer-script written by shellex_preexec already do that?

In any case, this isn't part of the escape-functionality, so it doesn't belong in this file.